### PR TITLE
Made it look pleasant (#8)

### PR DESCRIPTION
**Pull request by aiecel**
Hi! I returned after a small break and did a little redesign of the sidebar. Lookin lovely IMO. Hope you'd like it too!

EPILEPSY WARNING
TRAINEE FRONTEND DEVELOPER CODE LURKING AHEAD

### CODE REVIEW

File: crap-advisor-frontend/src/core/components/Map/index.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import "./index.css";
-import {MapContainer, TileLayer, ZoomControl} from "react-leaflet";
+import {MapContainer, TileLayer, ZoomControl, useMap} from "react-leaflet";
```
Comment (akichula) [inline crap-advisor-frontend/src/core/components/Map/index.tsx:3]: Remove unused "useMap" function

File: crap-advisor-frontend/src/core/components/RestroomMarker/index.tsx [hunk 2]
```diff
@@ -1,27 +1,44 @@
...
+
 interface IRestroomMakerProps extends Pick<IMapProps, 'selectRestroom'> {
-    restroom: Restroom;
+    restroom: Restroom
 }
 
 const RestroomMarker = ({restroom, selectRestroom}: IRestroomMakerProps) => {
+
+    const iconImages = [restroom1, restroom2, restroom3, restroom4, restroom5]
+    let currentImage = restroom4
```
Comment (akichula) [inline crap-advisor-frontend/src/core/components/RestroomMarker/index.tsx:22]: I'm not sure that is a good realisation. Worth a look at the useState hook, where state will be is currentImage variable.

File: crap-advisor-frontend/src/core/components/Sidebar/index.css [hunk 3]
```diff
@@ -1,27 +1,76 @@
...
+}
+
+.welcome {
+    font-size: 16px;
+    font-weight: 400;
+    text-align: center;
+}
+
+.section-restroom {
+    --hue: 200;
```
Comment (akichula) [inline crap-advisor-frontend/src/core/components/Sidebar/index.css:37]: As good practice, you should create a main css file, where you can add global variables (for exaple, in `#root` element)

Comment (akichula) [reply crap-advisor-frontend/src/core/components/Sidebar/index.css:37]: My previous comment is apply to every HEX in your css code too. You can stay variables inside main.css file or create globals.css file

**Review by akichula — COMMENTED** [review]
Hey hey @aiecel. I left some coments, please check this out

File: crap-advisor-frontend/src/core/components/Map/index.tsx (hunk 1 above)
Comment (aiecel) [reply crap-advisor-frontend/src/core/components/Map/index.tsx:3]: My bad

File: crap-advisor-frontend/src/core/components/RestroomMarker/index.tsx [hunk 4]
```diff
@@ -1,27 +1,44 @@
...
+import restroom3 from "../../../assets/restroom_4.png";
+import restroom4 from "../../../assets/restroom_4.png";
+import restroom5 from "../../../assets/restroom_5.png";
+
 interface IRestroomMakerProps extends Pick<IMapProps, 'selectRestroom'> {
-    restroom: Restroom;
+    restroom: Restroom
 }
 
 const RestroomMarker = ({restroom, selectRestroom}: IRestroomMakerProps) => {
```
[the remark this answers is not in the document]
Comment (aiecel) [reply crap-advisor-frontend/src/core/components/RestroomMarker/index.tsx:19]: What is the purpose of this?

File: crap-advisor-frontend/src/core/components/RestroomMarker/index.tsx (hunk 2 above)
Comment (aiecel) [reply crap-advisor-frontend/src/core/components/RestroomMarker/index.tsx:22]: Ok, I think i need to deep dive into those so-called "hooks"

File: crap-advisor-frontend/src/core/components/RestroomMarker/index.tsx [hunk 5]
```diff
@@ -1,27 +1,44 @@
...
+
+    const iconImages = [restroom1, restroom2, restroom3, restroom4, restroom5]
+    let currentImage = restroom4
+
+    if (restroom.rating) {
```
[the remark this answers is not in the document]
Comment (aiecel) [reply crap-advisor-frontend/src/core/components/RestroomMarker/index.tsx:24]: Add restroom.rating to what???

File: crap-advisor-frontend/src/core/pages/MainPage/index.css [hunk 6]
```diff
@@ -1,6 +1,9 @@
+@import url('https://fonts.googleapis.com/css2?family=Literata:opsz,wght@7..72,300;7..72,400;7..72,500;7..72,700&family=Prata&display=swap');
```
[the remark this answers is not in the document]
Comment (aiecel) [reply crap-advisor-frontend/src/core/pages/MainPage/index.css:1]: "Nods"

File: crap-advisor-frontend/src/core/components/Sidebar/index.css (hunk 3 above)
Comment (aiecel) [reply crap-advisor-frontend/src/core/components/Sidebar/index.css:37]: I've moved some styles to main.css as you said

File: crap-advisor-frontend/src/core/pages/MainPage/index.css [hunk 7]
```diff
@@ -1,6 +1,9 @@
...
+
 .mainContainer {
-    font-family: "Arial", serif;
-    display: flex;
+    font-family: "Literata", sans-serif;
+    display: grid;
```
[the remark this answers is not in the document]
Comment (aiecel) [reply crap-advisor-frontend/src/core/pages/MainPage/index.css:5]: We (you) will be reworking the sidebar anyway, so let's leave this like that for now

File: crap-advisor-frontend/src/core/components/Map/index.tsx [hunk 8]
```diff
@@ -1,25 +1,24 @@
...
 const Map = ({selectRestroom}: IMapProps): JSX.Element => {
-    const defaultPosition: LatLngExpression = [53.21176, 50.18394];
-    const defaultZoom: number = 13;
     const {state: {data: restrooms}} = useDataApi<Restroom>(ApiEndpoint.GetAllRestrooms, [], 'GET');
 
     return (
         <MapContainer
             className="map"
-            center={defaultPosition}
+            center={toLatLngExpression(defaultLocation)}
```
Comment (akichula) [inline crap-advisor-frontend/src/core/components/Map/index.tsx:21]: I'm really like this elegant solution, applause

**Review by akichula — APPROVED** [review]
@aiecel Now you can merge

**Pull request merged**